.github/workflows/outdated_upgrade.yml

```diff
@@ -52,4 +52,4 @@ jobs:
 
     - name: Testing...
       run: |
-        make command-outdated-upgrade
+        npm run test-unsafe test/jest/outdated.test.js
```

Comment:
The name should be `outdated-upgrade.test.js`?

Reply:
correct, renamed